test/quantization/core/test_quantized_tensor.py

```diff
@@ -267,23 +267,30 @@ def test_qtensor_legacy_new_failure(self):
         self.assertRaises(RuntimeError, lambda: qr.new(torch.Size([2, 3])))
         self.assertRaises(RuntimeError, lambda: qr.new([6]))
 
-    def test_per_channel_qtensor_creation(self):
+    def test_per_channel_qtensor_creation_cpu(self):
+        self._test_per_channel_qtensor_creation(torch.device('cpu'))
+
+    @unittest.skipIf(not TEST_CUDA, "No gpu is not available.")
```

Comment:
nit: Should be No gpu is available?

Reply:
another fix which is only applied at the next PR in the stack, let me fix that.

Reply:
fixed